### ISSUE TITLE
[Feature Request] Enable "Boost" part of Nvidia Reflex

### ISSUE THREAD

**Issue by Foul-Tarnished** [shadps4-emu/shadPS4#1036]
[SpecialK](https://github.com/SpecialKO/SpecialK) adds nvidia reflex to any d3d11/d3d12 game, but for Vulkan it can only enable the "boost" part of reflex, which should still be useful for better framepacing

(iirc the boost part stop cpu and gpu from throttling their frequency, which could happen if their usage is not constant and not that high)

**Comment by MaxMo20** [comment]
Can't we do this with RTSS\MSI Afterburner? Its in beta but it does work but I think I heard that it affects frame time more so then it does with controller latency, less so. But maybe after their updates it may improve control latency even more.

**Comment by Foul-Tarnished** [comment]
> Can't we do this with RTSS\MSI Afterburner? Its in beta but it does work but I think I heard that it affects frame time more so then it does with controller latency, less so. But maybe after their updates it may improve control latency even more.

Not afterburner, but probably with rtss, like SpecialK.

But the idea is not to require a 3rd party app, tonbe launched and hooked everytime...

**Comment by MaxMo20** [comment]
Yes I know, usually though most install both hence the /. I see what you mean though. A workaround is to tell RTSS to open everytime you log in/turn on.
